src/app/modules/+badges/badges-page.module.ts

```diff
@@ -1,8 +1,8 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
-import { MaterialModule } from 'src/app/material/material.module';
-import { SharedModule } from '../shared/shared.module';
+import { MaterialModule } from 'src/app/shared/modules/material/material.module';
+import { SharedModule } from '../../shared/shared.module';
```

Comment:
I will recommend to use relative paths, not absolute.

Reply:
If i understood well, reletive path like below one, by the way MaterialModule is unnesessary.